zeppelin/new_converter.py

```diff
@@ -0,0 +1,92 @@
+import re
+import os
+import base64
+from .converter import ZeppelinConverter
+
+
+class NewConverter(ZeppelinConverter):
+    """NewConverter converts Zeppelin version 0.7.1 notebooks to Markdown."""
+
+    MD_EXT = '.md'
```

Comment:
This should probably go in the base class. Actually, what about renaming the base class MarkdownConverter? The `zeppelin` part can be omitted since that's already the module name.